.gitignore

```diff
@@ -27,7 +23,6 @@ package-lock.json
 target/
 dist/
 src/types
-project.yaml
```

Comment:
are you sure you want to remove this @stepanLav ?

Reply:
yeah without that file, codegen will fall down:

<img width="663" alt="Screenshot 2024-04-29 at 12 25 08" src="https://github.com/novasamatech/subquery-proxy/assets/40560660/ca30b54d-3db5-42bd-9365-71351f268e11">

Reply:
yeah without that file, codegen will fall down:

<img width="663" alt="Screenshot 2024-04-29 at 12 25 08" src="https://github.com/novasamatech/subquery-proxy/assets/40560660/ca30b54d-3db5-42bd-9365-71351f268e11">

Reply:
I know it will, that's why I mention in the Readme:
> Copy one of the chain files into project.yaml: e.g for polkadot cp polkadot.yaml project.yaml

I just don't understand what was your process before.

Reply:
that not necessary, we are using -f flag to specify which project should be used